packages/design-system/src/scalars/components/amount-field/amount-field.tsx

```diff
@@ -167,6 +167,7 @@ const AmountFieldRaw: FC<AmountFieldProps> = ({
               disabled={disabled}
               onChange={handleOnChangeSelect}
               options={options}
+              className="flex h-10"
```

Comment:
the right height should be ensured in the `SelectField` component

Reply:
you are right, done